os/android/iahwc2.cpp

```diff
@@ -978,6 +978,19 @@ HWC2::Error IAHWC2::HwcDisplay::ValidateDisplay(uint32_t *num_types,
 #ifdef FORCE_ALL_DEVICE_TYPE
   force_all_device_type = true;
 #endif
+
```

Comment:
pls add some comments here why this w/a code piece is needed otherwise it looks weird.

Reply:
Updated